comfy_cli/command/run.py

```
@@ -283,87 +780,204 @@ def log_node(self, type, node_id):
+
+        node_id = data["node"]
+        if self.current_node:
+            self.remaining_nodes.discard(self.current_node)
+            self.update_overall_progress()
+        self.current_node = node_id
+        self.log_node("Executing", node_id)
+        if self.emitter.json_mode:
+            self.emitter.emit_node_executing(node_id)
         return True
 
     def on_cached(self, data):
```

Pylint (4.0.5)

[convention] Missing function or method docstring

(C0116)